wp-content/themes/reconasia/assets/_scss/blocks/post/_members.scss

```
@@ -0,0 +1,72 @@
+@use '../abstracts' as *;
+
+.wp-block-lazyblock-team-members {
+  a:not([class]) {
+    max-width: fit-content;
+    color: $color-link-primary-200;
+    @extend %text-style-paragraph-medium-short-caps;
+
+    &:hover {
+      color: $color-bg-dark-100;
+    }
+
+    &:focus {
+      color: $color-bg-dark-100;
+      background: $color-bg-light-300;
+      outline: 0;
+    }
+
+    .icon {
+      font-size: 0.8em;
+    }
+  }
+
+  .members__list--no-image {
+    margin: rem(8) 0 rem(40);
+
+    @include breakpoint('small') {
+      column-count: 2;
+      column-gap: rem(48);
+    }
+
+    /* stylelint-disable-next-line */
+    .members__item + .members__item {
+      margin-top: rem(16);
+    }
+  }
+
+  .members__list--has-image {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(#{rem(260)}, 1fr));
+    margin: rem(16) 0 rem(16);
```

It looks like the top margin should be `rem(8)`. 
```suggestion
    margin: rem(8) 0 rem(16);
```